### ISSUE TITLE
UX/UI : add a link to pricing 

### ISSUE THREAD

**Issue by superpositif** [Instant-Visio/InstantVisio-WebApp#603]
Je me connecte pour la première fois. Le BO s'ouvre. J'ai accès à toutes les fonctionnalités mais pas de crédits. On pourrait ajouter un lien vers la page pricing via un bouton "acheter des crédits".

**Comment by MattiaPrimavera** [comment]
De ma compréhension on n'a pas la notion de *credits* pour le moment, mais il faudrait expliquer que l'on ne pourra pas utiliser le backoffice avant de payer un abonnement. Plusieurs possibilités:
- rédiriger automatiquement les utilisateurs premium qui n'ont pas acheté un abonnement directement vers la page pricing au login
- ajouter un bandeau pour expliquer que tt fonctionnalité premium est désactivé, et qu'il faut souscrire à un abonnement (avec lien)

**Comment by superpositif** [comment]
Oui, en l'état c'est pas forcément hyper compréhensible, on peut se demander ce qu'on fait là, et pourquoi l'interface ne fonctionne pas. Mais dans l'idée je trouve ça assez cool de pouvoir tester le formulaire et voir l'interface. ça peut donner envie
